### PR TITLE
Added ELO Rating System Doc

### PR DESCRIPTION
**Pull request by simoninithomas**
Hello,

### Proposed change(s)

- I wrote a small doc about ELO rating system, I think it can help people who want to build multi-agents environments in ML-Agents to grasp rapidly what is ELO.

- The first part is directly taken from your Blogpost, I added the second that explains ELO with mathematical formulas and a simple example.

Have a nice day,

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [X] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

**Comment by miguelalonsojr** [comment]
hey @simoninithomas , thanks for the contribution! Can you make sure to rebase/merge main, resolve conflicts, and push? That should trigger our latest CI/CD. Once you get to that point, can you update the proposed documentation for the latest version on main? Thanks again!

**Review by taozhuo — CHANGES_REQUESTED** [review]
Just the empty lines. We are ready to approve and accept it.

File: docs/ELO-Rating-System.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# ELO Rating System
+
```
[the remark this answers is not in the document]
Comment (miguelalonsojr) [reply docs/ELO-Rating-System.md:2]: @simoninithomas Can you update address this comment? I can merge this in this week once the comment has been addressed.

**Comment by simoninithomas** [comment]
I updated based on what you asked ✅. Sorry for the big delay I was cleaning the notifications today and I saw this one.

**Comment by simoninithomas** [comment]
Hey there, I updated the doc: removed tennis illustration and added the link in ML-Agents-Overview.md#Training in Competitive Multi-Agent Environments with Self-Play

**Pull request merged**